src/main/java/CommandParser.java

```diff
@@ -0,0 +1,43 @@
+public class CommandParser {
+
+    public static String getCommand(String input) {
+        return input.split(" ")[0];
+    }
+
+    public static String getDeadlineTaskDescription(String input) {
+        int firstSpaceIndex = input.indexOf(" ");
+        int slashIndex = input.indexOf("/by");
+        // +1 to exclude " " and -1 to exclude "/"
```

Comment:
Good use of comments so that people reading your code can understand your thinking.